docs/Clearwater_Configuration_Options_Reference.md

```diff
@@ -110,6 +110,8 @@ This section describes optional configuration options, particularly for ensuring
 
 * `homestead_provisioning_port` - the HTTP port the Homestead provisioning interface listens on. Defaults to 8889. Not needed when using an external HSS.
 * `sas_server` - the IP address or hostname of your Metaswitch Service Assurance Server for call logging and troubleshooting. Optional.
+* `snmp_ip` - this determines where SNMP alarms get sent, and is a comma-separated list of IP addresses (e.g. `snmp_ip=0.0.0.0,0.0.0.1`).
+* `snmp_notification_types` - this determines what format SNMP alarms are sent in - possible values are `snmp_notification_types=rfc3877` (the default), `snmp_notification_types=enterprise`, or `snmp_notification_types=rfc3877,enterprise` to send both formats. See the [SNMP alarms documentation](SNMP_Alarms.md) for information about the difference.
```

Comment:
This implies that `snmp_notification_types=enterprise,rfc3877` isn't valid. Maybe easier to read as something like "a comma separated list of SNMP alarm formats. Valid alarm formats are ..."